server/hooks.go

```diff
@@ -80,11 +111,18 @@ func (p *Plugin) UserHasJoinedChannel(c *plugin.Context, channelMember *model.Ch
 		)
 	}
 
-	postChannel := &model.Post{
-		UserId:    p.botUserID,
-		ChannelId: channelMember.ChannelId,
-		Message:   string(data),
-	}
-	time.Sleep(1 * time.Second)
-	_ = p.API.SendEphemeralPost(channelMember.UserId, postChannel)
+	// Commented out in case we do not want the user immediately greeted with an ephemeral message
+	// postChannel := &model.Post{
+	// 	UserId:    p.botUserID,
+	// 	ChannelId: channelMember.ChannelId,
+	// 	Message:   string(data),
+	// }
+
+	// if actor.Id == channelMember.UserId {
+	// 	time.Sleep(3 * time.Second)
+	// } else {
+	// 	time.Sleep(15 * time.Second)
+	// }
+
+	// _ = p.API.SendEphemeralPost(channelMember.UserId, postChannel)
```

Comment:
Why does this PR touch this code? Is it related to the changes you are doing here?